impeller/aiks/aiks_unittests.cc

```diff
@@ -2943,13 +2943,15 @@ TEST_P(AiksTest, CanRenderForegroundAdvancedBlendWithMaskBlur) {
 
 // Regression test for https://github.com/flutter/flutter/issues/126701 .
 TEST_P(AiksTest, CanRenderClippedRuntimeEffects) {
-  if (GetParam() != PlaygroundBackend::kMetal) {
+  if (GetParam() == PlaygroundBackend::kVulkan) {
```

Comment:
Perhaps move this to `PlaygroundTest::BackendSupportsFragmentProgram`?

Reply:
Done